rllib/env/multi_agent_env_runner.py

```diff
@@ -414,17 +414,16 @@ def _sample_episodes(
 
         done_episodes_to_return: List[MultiAgentEpisode] = []
 
-        # Reset the environment.
-        # TODO (simon): Check, if we need here the seed from the config.
-        obs, infos = self.env.reset()
-
         # Create a new multi-agent episode.
         _episode = self._new_episode()
         self._make_on_episode_callback("on_episode_created", _episode)
         _shared_data = {
             "agent_to_module_mapping_fn": self.config.policy_mapping_fn,
         }
 
+        # Reset the environment.
```

Comment:
Remove from this PR. I think it belongs to another PR?

Reply:
What exactly do you mean? The comment was there before.